libs/damap/src/lib/components/dmp/doc-data-quality/doc-data-quality.component.css

```
@@ -6,3 +6,9 @@
 .full-width {
   width: 100%;
 }
+
+.customTooltip {
```

```suggestion
.custom-tooltip {
```

also replace at other locations